tools/test.py

```diff
@@ -1349,8 +1349,7 @@ def ReadConfigurationInto(path, sections, defs):
     if prefix_match:
       prefix = SplitPath(prefix_match.group(1).strip())
       continue
-    print "Malformed line: '%s'." % line
-    return False
+    raise Exception("Malformed line: '%s'." % line)
   return True
```

Comment:
You can drop the `return True` now, none of the testcfg.py scripts check its return value.